tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -617,11 +618,6 @@ class MklConcatOp : public OpKernel {
       if (concat_dim < 0) concat_dim = expected_dims + concat_dim;
 
       for (auto& s : input_shapes) {
```

Comment:
So GetMklShapeList() can return a mix of MKL and TF tensors? If not,  perhaps name the array mkl_input_shapes for clarity?

Reply:
Agree. Done with renaming - multiple lines of code changes in related functions.